environment/cpu/src/cloud/benchflow/monitors/cpu.go

```diff
@@ -180,6 +180,41 @@ func deltaHandler(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 }
+*/
+
+func dataHandler(w http.ResponseWriter, r *http.Request) {
+	if !monitoring {
+		fmt.Fprintf(w, "Currently not Monitoring")
+		return
+	}
+	query := r.FormValue("select")
+	if query == "all" {
+		for _, each := range containers {
+			if each.ID != "" {
+				js, err := json.Marshal(each)
+				if err != nil {
+					http.Error(w, err.Error(), http.StatusInternalServerError)
+				    return
+			    }
+			    w.Header().Set("Content-Type", "application/json")
+			    w.Write(js)
```

Comment:
You should build the JSON representing the response, and then write it only once, after you `json.Marshal` it.